src/OpenLoco/src/Vehicles/Routing.cpp

```diff
@@ -130,7 +145,7 @@ namespace OpenLoco::Vehicles
             return ((((interest.loc.x ^ interest.loc.z) / 32) ^ interest.loc.y) ^ interest.trackAndDirection) & mapSizeMask;
         }
 
-        // 0x004A38DE
+        // 0x004A38DE & 0x004A3972
```

Comment:
i think i recall you guys said something about zehmatt's tool can't parse these? can you add a second comment line with the second address as an easy fix?